qa/test_template.sh

```diff
@@ -8,8 +8,13 @@ set -x
 topdir=$(cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null && pwd )/..
 source $topdir/qa/setup_test_common.sh
 
+# Set runner for python tests
+python_test_runner_package="nose"
+python_test_runner="python -m nose"
+python_test_args="--verbose -s"
+
```

Comment:
Setting Python testing framework. First is the name of pip package to install, second is a command to run the test, third are args that we want to pass to all tests.